aws-android-sdk-appsync-gradle-plugin/build.gradle

```diff
@@ -13,11 +13,11 @@ dependencies {
 
     compileOnly gradleApi()
     compileOnly localGroovy()
-    compile "com.moowork.gradle:gradle-node-plugin:1.0.0" // compileOnly
-    compile 'com.android.tools.build:gradle:3.2.1' // compileOnly
-    compile 'com.squareup.moshi:moshi:1.5.0' // impl
+    implementation "com.moowork.gradle:gradle-node-plugin:1.0.0" // compileOnly
```

Comment:
Why are the dependencies marked with compileOnly using implementation? Check the published POMs for guidance.